code/modules/admin/verbs/randomverbs.dm

```diff
@@ -549,6 +549,32 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	admin_ticket_log(M, msg)
 	SSblackbox.record_feedback("tally", "admin_verb", 1, "Rejuvinate") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+/client/proc/cmd_admin_freeze(mob/living/M in GLOB.mob_list)
+	set category = "Special Verbs"
+	set name = "Freeze Player"
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.")
+		return
+	if(!istype(M))
+		return
+
+	if(!M.adminfrozen)
+		var/sleepies = M.AmountSleeping()
+		M.adminfrozen = sleepies ? sleepies : 1
+		M.SetSleeping(200000)//20k seconds to get your admin shit together
```

Comment:
Why don't use timer here?
```suggestion
		M.SetSleeping(200000 SECONDS)//20k seconds to get your admin shit together
```

Reply:
What's the difference?

Reply:
One is byond built in, second uses timer subsystem.
This feels wrong anyway, if admin freezes you he has no intention of you waiting 20000 seconds, you should not put this on any timer just freeze and unfreeze verbs.

Reply:
> One is byond built in, second uses timer subsystem.
> This feels wrong anyway, if admin freezes you he has no intention of you waiting 20000 seconds, you should not put this on any timer just freeze and unfreeze verbs.

Using the freeze verb again unfreezes them....